src/SelfHelp/README.md

```diff
@@ -0,0 +1,69 @@
+<!-- region Generated -->
+# Az.SelfHelp
+This directory contains the PowerShell module for the SelfHelp service.
+
+---
+## Status
+[![Az.SelfHelp](https://img.shields.io/powershellgallery/v/Az.SelfHelp.svg?style=flat-square&label=Az.SelfHelp "Az.SelfHelp")](https://www.powershellgallery.com/packages/Az.SelfHelp/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.SelfHelp`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+```yaml
+# pin the swagger version by using the commit id instead of branch name
+branch: f9d72b807dd04fb7c606e9810bfdb9ad6a74b91e
+require:
+  # readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../readme.azure.noprofile.md
+  #  - $(repo)/specification/xxx/resource-manager/readme.md
+  # If the swagger has not been put in the repo, you may uncomment the following line and refer to it locally
+  - $(this-folder)/azure-rest-api-specs/specification/help/resource-manager/readme.md
```

Comment:
The failure in CI is due to this file is missing. Please check it in.

Reply:
Updated swagger to point to the latest stable version that is checked in.